src/SDKs/CognitiveServices/CognitiveServices.Tests/CognitiveServices.Tests.csproj

```diff
@@ -27,4 +27,13 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
@shahabhijeet,
1. Removed those references. They were automatically added when I updated them in NuGet manager.
2. Add Microsoft.Azure.Management.ResourceManager 1.6.0-preview reference and changed UT code a little bit accordingly.
3. Re-recorded all unit tests.
4. Removed that #DNX451 constant.